icicle/include/vec_ops/vec_ops.cuh

```diff
@@ -42,6 +44,7 @@ namespace vec_ops {
       false, // is_b_on_device
       false, // is_result_on_device
       false, // is_async
+      false, // is_in_montgomery_form
```

Comment:
Perhaps it would be better to separate the Montgomery form flag for inputs and the result.

Reply:
In others we let the user specify it per input. For example 
https://github.com/ingonyama-zk/icicle/blob/621676bd41659bc101f29dc7ac54f29c7cdaa92b/icicle/include/msm/msm.cuh#L67